apps/desktop/src/lib/branch/BranchLaneContextMenu.svelte

```diff
@@ -15,10 +15,14 @@
 	import Button from '@gitbutler/ui/inputs/Button.svelte';
 	import Modal from '@gitbutler/ui/modal/Modal.svelte';
 
-	export let contextMenuEl: ContextMenu;
-	export let target: HTMLElement;
-	export let onCollapse: () => void;
-	export let onGenerateBranchName: () => void;
+	interface Props {
+		contextMenuEl?: ContextMenu;
```

Comment:
Since the component is only used in one place, do we need to make this optional?

Reply:
Yeah I think we do unfortunately because the `contextMenu` variable from the parent is `undefined` initially and only gets bound once the child (`ContextMenu`) of the `BranchLaneContextMenu` is rendered / mounted, right?